File: program/find-area-of-a-cuboid/find_area_of_a_cuboid.go
package main

import "fmt"

func main() {
	var l, b, h int

	fmt.Printf("Enter length of Cuboid:\t")
	fmt.Scanf("%d", &l)

	fmt.Printf("Enter width of Cuboid:\t")
	fmt.Scanf("%d", &b)

	fmt.Printf("Enter height of Cuboid:\t")
	fmt.Scanf("%d", &h)

	fmt.Println("Area:", 2*(l*b+b*h+l*h))
}

File: program/find-area-of-a-cuboid/FindAreaOfACuboid.java
import java.util.Scanner;

public class findareaofacuboid {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Length : ");
        int len = input.nextInt();
        System.out.print("Width : ");
        int wid = input.nextInt();
        System.out.print("Height : ");
        int ht = input.nextInt();
        System.out.println();
        System.out.print("Area : ");
        int area = 2*(len*wid+wid*ht+ht*len);
        System.out.println(area);
    }
}
